docs/Running-Mastodon/Production-guide.md

```diff
@@ -34,9 +34,17 @@ server {
   keepalive_timeout    70;
   sendfile             on;
   client_max_body_size 0;
-  gzip off;
 
-  root /home/mastodon/live/public;
```

Comment:
When I tried it, Got no CSS, no JS and no pictures, all in 404. (folder doesn't mapped correctly in /public/ folder.) 

Work great without this directive,

(maybe it's my fault).